lib/main.js

```diff
@@ -83,12 +82,11 @@ const fixPHPCSColumn = (lineText, givenCol, tabWidth, currentStandards, version)
     } else {
       screenCol += 1;
     }
-    if (screenCol >= column) {
-      column = col + 1;
-      break;
+    if (screenCol >= givenCol) {
+      return col + 1;
     }
   }
-  return column;
+  return lineText.length + 1;
```

Comment:
Isn't this going to be wrong on every line that _doesn't_ have whitespace on the end?

Reply:
This will set the error at the end of the line with a length of 0 rather than the last character of the line. If the last character is supposed to be the error it will catch on the return above this.

Reply:
The reason this fix is needed is because every line passed to `fixPHPCSColumn` won't have a line ending.

The text that is sent to `fixPHPCSColumn` is from: `const lineText = textEditor.getBuffer().lineForRow(line);`

and `lineForRow` returns: [source](https://atom.io/docs/api/v1.34.0/TextBuffer#instance-lineForRow)
> the text of the line at the given 0-indexed row, without its line ending.

If the error is on the line ending it will be passed `lineText.length`